### PR TITLE
journal-week-3

### PR DESCRIPTION
**Pull request by jeffworley**

### CODE REVIEW

**Comment by jmcreasman** [comment]
It's a start! :+1:

**Comment by rosalsm** [comment]
Keep being so ooooptimistic! :+1:

**Comment by s4dmw** [comment]
dude...keep this up and you'll end up with your picture in the definition of tl;dr on urban dictionary...

looks good but i can't wait to see if you'll be able to write a tutorial that isnt a "465 page thumper"

:+1: :beer:

**Comment by ShayHall** [comment]
You're voice as a writer is light and upbeat.  It makes learning easier and more funner.  :+1: Things are looking :point_up: .

**Comment by jeremyrist** [comment]
:+1: Great issue man! Learned so much on there, I second Shay on that one...you're a good writer. I'm lucky I checked yours out, I couldn't figure out where the %$&#\* `package.json` file was haha. And great journal I am totally sold on the idea of _RESPONSIVE WEB DESIGN_

**Comment by boshnivolo** [comment]
Your tutorial looks like it's on its way to being super informative! Keep it up :+1:

File: tutorial-week-3.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Rough Draft
+
+Oh hello.  I thought we just spoke in my other journal entry.  You know the one with the oh so clever and creative title of journal-week-3.md?  That's what brought you over here?  Then I had better deliver right?!?!  With that massive expectation set, I shall proceed to underwhelm.  This weeks tutorial will be on Google Chrome Developer Tools.  Note, this will not be a 465 page thumper will details about every single feature but rather it will be an overview of what it is, how you can access it, why you might want to use it, and why I have fallen truly, madly, deeply in love with it over the last week.  I know the 'Savage Garden' reference was painful for me too.  I have no clue why that song popped into my head.
```
Comment (kellymurray) [inline tutorial-week-3.md:3]: not savage garden!!

**Pull request merged**